include/seqan3/alphabet/nucleotide/nucleotide_base.hpp

```diff
@@ -29,10 +30,11 @@ namespace seqan3
  * You can use this class to define your own nucleotide alphabet, but types are not required to be based on it to model
  * seqan3::nucleotide_concept, it is purely a way to avoid code duplication.
  *
- * The derived type needs to define only the following table as static member variable:
+ * The derived type needs to define the following static member variables (can be private):
  *
  *   * `static std::array<THAT_TYPE, value_size> complement_table` that defines for every possible rank value
  *     the corresponding complement.
+ *   * `static constexpr auto char_validator`: a functor that is offers operator(char c) and return bool.
```

Comment:
do they? I don't see the change in the respective nucleotide alphabets.

Reply:
good catch, this changed in between revisions of my PR